sdk/tests/MCPClientManager.server-mgmt.test.ts

```
@@ -13,43 +14,32 @@ describe("MCPClientManager", () => {
-      });
+      await manager.connectToServer("myserver", everythingServerConfig());
 
       expect(manager.hasServer("myserver")).toBe(true);
       expect(manager.hasServer("nonexistent")).toBe(false);
     }, 30000);
 
     it("should get server config", async () => {
-      await manager.connectToServer("configured", {
-        command: "npx",
-        args: ["-y", "@modelcontextprotocol/server-everything"],
+      await manager.connectToServer("configured", everythingServerConfig({
```

ESLint

[error] Replace `"configured",` with `⏎········"configured",⏎·······`

(prettier/prettier)